src/features/encoders/avif/client/index.tsx

```diff
@@ -67,18 +75,18 @@ export class Options extends Component<Props, State> {
     return {
       options,
       lossless,
-      quality: maxQuant - cqLevel,
+      quality: cqLevel,
       separateAlpha,
-      alphaQuality:
-        maxQuant -
-        (separateAlpha ? options.cqAlphaLevel : defaultOptions.cqLevel),
+      alphaQuality: separateAlpha
+        ? options.cqAlphaLevel
+        : defaultOptions.cqLevel,
```

Comment:
In line 84:

```
        options.subsample === 0 || lossless
```

Is the `options.subsample === 0` check correct? `options.subsample === 0` means 4:0:0 (monochrome). I wonder if the check should be `options.subsample === -1`.

Also, if `lossless` is true, it seems that `subsample` should be set to 3 (4:4:4).

Reply:
Hmm, it turns out we aren't even exposing options 0 and 2 in the form. My guess is it handles the default value (0) before the user selects anything? I'll look into it

Reply:
I wonder if the default value of `options.subsample` should be something other than 0 (e.g., 1 = 4:2:0), or an invalid value such as -1 to indicate that it is not initialized.